src/common/sendpacket.c

```diff
@@ -671,6 +686,9 @@ sendpacket_get_hwaddr(sendpacket_t *sp)
 
     if (sp->handle_type == SP_TYPE_KHIAL) {
         addr = sendpacket_get_hwaddr_khial(sp);
+    } else if( sp->handle_type == SP_TYPE_LIBPCAP_DUMP) {
+        sendpacket_seterr(sp, "Error: sendpacket_get_hwaddr() not yet supported for pcap dump");
```

Comment:
This will need to get implemented.

Reply:
I think  sendpacket_get_hwaddr() only makes sense for real interfaces.  Pcap file as an output doesn't support sendpacket_get_hwaddr.

Reply:
OK, I still have this in staging and will investigate. I'll probably just change the error message.